spec/system/admin/reports_spec.rb

```diff
@@ -499,12 +499,16 @@
   end
 
   describe 'bulk coop report' do
+    let!(:order) { create(:completed_order_with_totals) }
+
     before do
       login_as_admin
       visit admin_reports_path
     end
 
     it "generating Bulk Co-op Supplier Report" do
+      pending("S2 bug #12768")
```

Comment:
I am wondering if should also assert if there is some data in the report ? Anyway I am going to look at this S2 bug shortly and will take care of it if needed.